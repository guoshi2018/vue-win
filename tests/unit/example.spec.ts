
import jestHelloWorld from "do-jest/jest-hello-world";
import jestMessageApp from "do-jest/jest-message-app";
import jestMessageList from "do-jest/jest-message-list";

describe('now testing all of vue-win', () => {
  jestHelloWorld();
  jestMessageApp();
  jestMessageList();
})
