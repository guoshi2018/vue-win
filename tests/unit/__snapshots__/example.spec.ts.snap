// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`now testing all of vue-win ,mount message app with its children: msgapp-1 1`] = `
<div id="message-app">
  <ul>
    <li>Cat</li>
    <li>Dog</li>
  </ul>
</div>
`;

exports[`now testing all of vue-win ,shallowMount message app without its children: msgapp-2 1`] = `
<div id="message-app">
  <message-list-stub messages="Cat,Dog"></message-list-stub>
</div>
`;

exports[`now testing all of vue-win ,test message-list vue alonely : message-list inner 1`] = `
<ul>
  <li>Cat</li>
  <li>Dog</li>
</ul>
`;
